atuin-client/config.toml

```diff
@@ -108,10 +117,25 @@
 ## start with ^ or end with $, they'll match anywhere in CWD.
 ## For details on the supported regular expression syntax, see
 ## https://docs.rs/regex/latest/regex/#syntax
-# cwd_filter = [
+# cwd = [
```

Comment:
`cmd` and `cwd` look very similar. Also, one doesn't have to write these more than once, thus I believe that the following are more descriptive (for non-experts):

`command` or `commands`
and
`directory` or `directories`